addons/image/tga.c

```diff
@@ -320,7 +338,9 @@ ALLEGRO_BITMAP *_al_load_tga_f(ALLEGRO_FILE *f, int flags)
 
       case 1:
          /* paletted image */
-         if ((palette_type != 1) || (bpp != 8)) {
+         /* Only support 8 bit palettes (up to 256 entries) though in principle the file format could have more(?)*/
+         if ((palette_type != 1) || (bpp != 8) || (palette_start + palette_colors) > 256) {
+            ALLEGRO_ERROR("Invalid palette/image/bpp combination %d/%d/%d.\n", image_type, palette_type, bpp);
```

Comment:
The values don't correspond to the description (e.g. you could say 'Invalid image/palette/bpp' instead).

Reply:
Good point -  I just copied it from line 357 / line 365 actually (`git blame` knows who wrote those lines :wink: ) - I'll fix them all up the same.

Reply:
Whoops! Thanks!

Reply:
done!